velox/connectors/hive/HiveDataSource.cpp

```diff
@@ -352,9 +356,14 @@ int64_t HiveDataSource::estimatedRowSize() {
 }
 
 vector_size_t HiveDataSource::evaluateRemainingFilter(RowVectorPtr& rowVector) {
-  auto filterStartMicros = getCurrentTimeMicro();
   filterRows_.resize(output_->size());
-
+  DecodedVector decoded;
+  SelectivityVector baseRows;
```

Comment:
Maybe declare these alongside the other fields like the other SelectivityVector filterRows_ under

// Reusable memory for remaining filter evaluation.

I remember allocating these can add latency, though LazyVector::ensureLoadedRows may allocate them on the fly for complex types, so it's probably not a big difference either way.